docs/Maintainers.md

```diff
@@ -0,0 +1,95 @@
+| Dave Thaler          | Microsoft | dthaler@microsoft.com         | dthaler        |
+| John Kordich         | Microsoft | johnkord@microsoft.com        | johnkord       |
+| Mike Brasher         | Microsoft | mikbras@microsoft.com         | mikbras        |
+| Simon Leet           | Microsoft | simon.leet@microsoft.com      | CodeMonkeyLeet |
+
+Committee Responsibilities
+--------------------------
+
+The primary responsibility of the Committee is to grant new committer rights
+(that is, write access to the main Open Enclave SDK repository or related
+repositories), and to grant new membership into the committee. Conversely, the
+Committee must also remove committer rights and membership from those found to
+be violating the project's Code of Conduct or otherwise negatively affecting the
+project's community health.
+
+This Committee is not intended to make every technical decision, as those should
+generally be made by agreement among committers as PRs are reviewed and merged.
+Where disagreements take place and need further resolution, those can be brought
+up with the Maintainers Committee as part of its responsibility to maintain the
+project's community health. Otherwise technical decisions are left to the active
+committers (by virtue of the liberal contribution policy).
+
+The Community Maintenance Committee should meet regularly, for example, once a
+month. This meeting is a private meeting among just the maintainers to nominate
+new committers and maintainer members. Priority consideration should be given to
+those actively contributing to the project. The Committee uses the consensus
+seeking process outlined above when making decisions, including adding or
+removing any members. The Committee should also discuss the community's health
+and work to resolve any negative issues.
+
+In order to maintain a healthy developer community, it is recommended that the
+Maintainers Committee also host a regular public community meeting. This meeting
+should be open all members of the community, and start with an open forum to
+hear questions or concerns from the community. Any remaining time in the meeting
+should be used to discuss and review open pull requests or issues (especially
+including design documents).
+
+Project Committers
+==================
+
+The following people have been granted commit permissions (that is, write
+access) to the Open Enclave SDK by the Community Maintenance Committee. The area
+column describes which technical areas each committer is most interested in, and
+therefore should usually be consulted for changes relating to that area.
+However, it is up to each committer to determine who should review which PR, and
+when to merge it. Remember that a PR must not be merged if a committer objects;
+instead, it should be brought up with the Community Maintenance Committee.
+
+| Name                  | Area                                      | GitHub Alias        |
```

Comment:
I think "General", "Core", and perhaps even "Security" are too broad and area not "Areas" per se.
Also from my experience with other open source projects, I would recommend having an area lead for "APIs and Language Bindings", and an area lead for "Platform Abstraction Layer (Host OS bindings and TEE bindings)".   And then allow them to designate a sub-maintainer per language binding, host OS binding, and TEE binding, if they want.

Reply:
I agree, I just couldn't think of better names at the time 😂